config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -124,27 +124,26 @@ find "$base" -name '*.yaml' -exec grep -q 'generated by gen-jobs.sh' '{}' \; -de
 # Resource usage of a job depends on whether it needs to build Kubernetes or not.
 resources_for_kubernetes () {
     local kubernetes="$1"
+    local indentation="$2"
 
     # "master" is always built from source. The experimental version might be built from
     # source, depending on what is in the prow.sh script. Here we play it safe and
     # request more resource when testing the experimental version, just in case.
+    echo "${indentation}resources:"
+    echo "${indentation} requests:"
```

Comment:
I think it's clearer to duplicate these two lines below, as it was before. For example, then it is obvious that `memory` and `cpu` are indented properly.

Reply:
done!